index.html

```diff
@@ -20,27 +22,160 @@
         <nav class="hamburger-menu"> <a href="#">☰</a></nav>
       </nav>
 
+      <section class="content">
+        <h1 class="title">
+          I'm Luis <br>Glad to see you!
+        </h1>
+        <div class="header-content">
+          <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through
+            some
+            of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+            contact me.</p>
+          <h2 class="connect"> <a href="#">Let's connect</a></h2>
+          <ul class="socials">
+            <li><a href="https://twitter.com/lujimhe" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/TwitterVector.svg" alt="Twitter logo"></a></li>
+            <li><a href="https://www.linkedin.com/in/lujim/" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
+            <li><a href="https://medium.com/@lu.jim" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/MediumVector.svg" alt="Medium logo"></a></li>
+            <li><a href="https://github.com/lu-jim" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/GithubVector.svg" alt="Github logo"></a></li>
+            <li><a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+                <img src="./resources/AngelListVector.svg" alt="AngelList logo"></a></li>
+          </ul>
+        </div>
+      </section>
+    </header>
+    <div class="works">
+      <article class="card-1">
+        <img src="./images/tonic1.png" alt="Tonic" class="project-image">
+
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-2">
```

Comment:
- [ ] Kindly fix the styling for the projects section for mobile version.
![Captura de pantalla_2021-09-19_00-04-28](https://user-images.githubusercontent.com/53111464/133916086-5060ff19-a95e-4379-b979-b82fae4ed27a.png)
![Captura de pantalla_2021-09-19_00-05-20](https://user-images.githubusercontent.com/53111464/133916104-64027e0d-5787-44fc-a459-7921f59715fa.png)

-  kindly read this article in order to learn how to use Chrome dev tools to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).